tests/integration/questionnaire/test_questionnaire_list_collector_primary_person.py

```diff
@@ -6,8 +6,13 @@
 
 
 class TestQuestionnaireListCollector(IntegrationTestCase):
-    def get_link(self, rowIndex, text):
-        selector = f"tbody:nth-child({rowIndex}) td:last-child a"
+    def get_previous_link(self):
+        selector = "#top-previous"
+        selected = self.getHtmlSoup().select(selector)
+        return selected[0].get("href")
```

Comment:
I can't see this being used (also, there is a `previous` method on `IntegrationTestCase`).